openpype/pipeline/create/subset_name.py

```diff
@@ -154,4 +158,10 @@ def get_subset_name(
         for key, value in dynamic_data.items():
             fill_pairs[key] = value
 
-    return template.format(**prepare_template_data(fill_pairs))
+    try:
+        return template.format(**prepare_template_data(fill_pairs))
+    except KeyError as exp:
+        error = "Value for '{}' key is missing in template '{}'.".format(
```

Comment:
```suggestion
        error = "Value for {} key is missing in template '{}'.".format(
```
Looking at your screenshot the reported value has double single quotes. Likely due to the KeyError's message value:
```python
a = {}
a["foo"]
# Error: 'foo'
# # Traceback (most recent call last):
# #   File "<maya console>", line 2, in <module>
# # KeyError: 'foo'
```
The string already includes single quotes.